Tests/IntegrationTests/Card Tests/CardComponentEventTests.swift

```diff
@@ -40,13 +40,12 @@ final class CardComponentEventTests: XCTestCase {
         let configDataDict = try XCTUnwrap(info?.configData?.stringOnlyDictionary)
         XCTAssertEqual(configDataDict["socialSecurityNumberMode"], "auto")
         XCTAssertEqual(configDataDict["hasInstallmentOptions"], "false")
-        XCTAssertEqual(configDataDict["showsStorePaymentMethodField"], "true")
         XCTAssertEqual(configDataDict["billingAddressRequired"], "true")
-        XCTAssertEqual(configDataDict["showsSecurityCodeField"], "true")
+        XCTAssertEqual(configDataDict["hideCVC"], "true")
```

Comment:
Maybe then also this needs to change to "false" (Not sure from the limited amount of code)